_posts/2022-09-30-chapter-tech-leads.markdown

```diff
@@ -0,0 +1,87 @@
+- is led by a tech lead and
+- has a size of max. 10 regular employees
+
+There are many definitions for what a Chapter Lead (CL) is and what is required to become one. For us, we defined it as a senior developer who can 
+- create a vision for technical topics,
+- mentor other developers, 
+- pass the passion for development to the members, and 
+- organizes joint efforts for education and training. 
+
+For that, the CL is free to organize the Chapter and communication as seems necessary.<br/>
+ He (there are only male leads in our case at the moment) would be responsible for 
+ - creating the vision for the Chapter
+ - setting up and maintaining  the Chapter Backlog
+ - for supporting team members in becoming experts in the Chapter's mission topic.
+
+A Chapter Lead would not decide on salary rises, approve vacation days or hire people. But a CL will always give valuable input and information regarding these topics to the line manager.
+
+## How did we do the setup?
+
+### 1. Cut the elephant into pieces
+Once the goal was decided, we had to figure out how to get there, and the first step was to cut out the chapters.
+Two approaches were discussed:
+- by technology, we would put Java devs in one squad, Kotlin devs in another, and those working on PKI topics in a third
+- by value stream, we would look at the recurring topics in most of our product teams, group them into categories (e.g., Cloud Services, Interoperability via FHIR/REST, Web Applications, etc.) and build the chapters around that
+We finally decided on the value stream approach as we thought it would help the product teams directly and also make the benefit of the system more transparent.
+
+### 2. Find the leads
+The next step was to find potential Chapter Leads. We knew we were looking for experienced developers that already stood out in discussions around technology and development culture and where we would trust that they would take it as a long-awaited chance to get things moving.
+I spoke to each candidate personally, and everyone was immediately on fire for the position, shooting ideas of what could be done and achieved in this setting.
+That showed we were on the right track! 
+Every candidate received a "First 100 days expectation"-email giving scope for the role. It is important to emphasize that the position was created as a tech lead. No disciplinary management tasks were involved. The message was: "That's your topic and your people - you are responsible for growing both with any support from management that it takes!"
+
+### 3. Start the work
+To create an environment for the chapter leads and developers where the education and growth of knowledge could bloom, we had to make a few yet fundamental decisions about the freedom of the Chapter. Some issues were easy as they were clearly defined by our company guidelines, e.g., who approves vacation requests. Others required deeper discussion like educational budget or flat rate time for chapter work.
```

Comment:
"few yet fundamental decisions about" => "few, yet fundamental, decisions about" (unsure, im not good at this...)

Reply:
"flat rate time" = "time allotment"

Reply:
The term "time allotment" misses the "blanket" character, what we discussed was a certain amount of time that is just reserved without any justification.